src/slash/drop.ts

```diff
@@ -32,7 +46,37 @@ export class DropCommand extends AutocompletableCommand {
 
 	protected override async execute(interaction: ChatInputCommandInteraction<"cached">): Promise<void> {
 		await interaction.deferReply();
-		const tournamentName = interaction.options.getString("tournament", true);
+		const tournamentName = interaction.options.getString("tournament");
+		if (!tournamentName) {
+			const players = await ManualParticipant.find({
+				where: { discordId: interaction.user.id },
+				relations: ["tournament"]
+			});
+
+			if (players.length < 1) {
+				await interaction.editReply(`You are not in any tournaments.`);
+				return;
+			}
+
+			if (players.length > 1) {
```

Comment:
Another check needs to be done (or the query altered) so that completed tournaments do not factor into this. Only care about preparing and in progress.

Reply:
Is this also true in the context command, then?

Reply:
Yes, drop from completed doesn't make sense